classes/submission/reviewRound/ReviewRound.inc.php

```diff
@@ -130,23 +132,26 @@ public function determineStatus() {
 		// Check if revisions requested or received, if this is latest review round and then check files
 		$roundStatus = $this->getStatus();
 
-		// If revisions have been requested, check to see if any have been
-		// submitted
-		if ($this->getStatus() == REVIEW_ROUND_STATUS_REVISIONS_REQUESTED || $this->getStatus() == REVIEW_ROUND_STATUS_REVISIONS_SUBMITTED) {
-			$submissionFileDao = DAORegistry::getDAO('SubmissionFileDAO');
-			$submissionFiles = $submissionFileDao->getRevisionsByReviewRound($this, SUBMISSION_FILE_REVIEW_REVISION);
-			if (empty($submissionFiles)) {
-				return REVIEW_ROUND_STATUS_REVISIONS_REQUESTED;
-			}
-			return REVIEW_ROUND_STATUS_REVISIONS_SUBMITTED;
-		}
+//		// If revisions have been requested, check to see if any have been
```

Comment:
Feel free to remove unnecessary code if you've implemented this a different way. Commented code clutters up the repo.